### DISCUSSION TITLE
New Vision API Results

### DISCUSSION THREAD

**Discussion by ecdye** [ecdye/macSup2Srt#25] [General]
@timj I just added support for the new Vision API in macOS 15. In my testing, I was noticing some interesting results for the results, where sometimes it appears that the new API is actually producing worse results. I would love your input on what you observe as you use it.

**Comment by timj** [comment]
Thanks. I had a quick try and the new API seems weirdly worse in many case. Trying ffmpeg / internal decoder / old API / new API you get different results every time and the feeling that if they were all combined taking the best of each you'd get the right answer...

I haven't tried PGS in a while but I imagine things are much better there because the images have more pixels? I still don't understand how copy and paste from the VobSub images is essentially perfect OCR...

**Comment by ecdye** [comment]
Yeah, I really don't either. Especially because as far as I can tell, the settings in the New API are the same as the old one. I can't image why it's so much worse. I also still don't understand why the copy paste is really accurate, but the programatic implementation is so far off of it. I'll keep trying to investigate. 

I'm hoping that Apple is still refining the new API and that it will be fixed in future updates.
